packages/app-bridge/src/types/Brandportal.ts

```diff
@@ -0,0 +1,20 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type BrandportalApi = {
+    i18n_settings: {
+        languages: BrandportalI18nLanguage[];
+    };
+};
+
+export type BrandportalI18nLanguage = {
+    language: string;
+    label: string;
+    default?: boolean;
+    draft?: boolean;
```

Comment:
Could those be defaulted to false instead of having them nullable?

Reply:
I changed the type accordingly.